runtime/doc/treesitter.txt

```diff
@@ -573,6 +573,15 @@ associated with patterns:
     • `injection.parent` - indicates that the captured node’s text should
       be parsed with the same language as the node's parent LanguageTree.
 
+Injection queries are currently run over the entire buffer, which can be slow
+for large buffers. To disable injections for a certain language, just place an
+empty query file in your 'runtimepath'. For example, to disable injections for
```

Comment:
```suggestion
query file containing only a comment in your 'runtimepath'. For example, to disable injections for
```

("empty" would mean "no contents", not even comments)

Reply:
That would still work.

Reply:
You mean adding an empty file? I was under the impression that it had to have _some_ contents and empty file was not enough. In that case, the documentation can (and should) be simplified significantly.

Reply:
The file just needs to exist to prevent other files being loaded. We can then use `extends` or `inherits` to pull in other files.

Reply:
OK, then that was a misunderstanding on my part; let's just document empty query then.